openfecwebapp/templates/legal_advisory_opinion_landing.html

```diff
@@ -0,0 +1,50 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/legal-search.html' as legal_search %}
+{% import 'macros/breadcrumbs.html' as breadcrumb %}
+
+{% set breadcrumb_links=[(cms_url + '/legal-resources', 'Legal resources')] %}
+
+{% block title %}
+  {% if query %}
+  Searching {{ document_type_display_name }} for &ldquo;{{ query }}&rdquo;
+  {% else %}
+  Search {{ document_type_display_name }}
+  {% endif %}
+{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  {{ breadcrumb.breadcrumbs(document_type_display_name, breadcrumb_links) }}
```

Comment:
We can hardcode `document_type_display_name` with "Advisory opinions" since it's never going to be anything else anymore.